### ISSUE TITLE
arpa/inet.h: No such file or directory

### ISSUE THREAD

**Issue by kbumsik** [RIOT-OS/RIOT#7721]
Hi,
When I try compiling `tests/driver_enc28j60` with the following command, 

> BOARD=stm32f4discovery USEMODULE+=gnrc_uhcpc make all

I got an error:

```
RIOT/sys/net/application_layer/uhcp/uhcp.c:11:23: fatal error: arpa/inet.h: No such file or directory
 #include <arpa/inet.h>
                       ^
compilation terminated.
```
Is it an unexpected error, or did I miss something? I would appreciate if someone let me know.
Thank you.

**Comment by miri64** [comment]
Can you try with `USEMODULE += posix`?

**Comment by miri64** [comment]
For me it is working with it, but there seems to be another dependency problem.

**Comment by miri64** [comment]
See https://github.com/RIOT-OS/RIOT/pull/7722 to fix that definitively.

**Comment by kbumsik** [comment]
It compiles correctly with your fix in #7722 .
Thank you for a quick response :+1:

**Comment by kbumsik** [comment]
I will close this issue since #7722  is merged.

**Comment by miri64** [comment]
Sorry we needed to revert that (see #7967), since that merge was against the GNRC merge-embargo. But it is fixed in #7925.
